docs/user/commercial/index.rst

```diff
@@ -15,24 +46,25 @@ Private repositories and private documentation
 Additional build resources
     Do you have a complicated build process that uses large amounts
     of CPU, memory, disk, or networking resources?
-    Our commercial offering has much higher default resources
-    that result in faster documentation build times
-    and we can increase it further for very demanding projects.
+    Commercial subscriptions offer more resources
+    which results in faster documentation build times.
+    Resources can be increased further for very demanding projects.
 
 Priority support
     We have a dedicated support team that responds to support requests during business hours.
-    If you need a quick turnaround, please signup for readthedocs.com.
 
 Advertising-free
     All commercially hosted documentation is always ad-free.
 
-.. _readthedocs.org: https://readthedocs.org
-.. _readthedocs.com: https://readthedocs.com
+Business features
+    A number of extra features exist specifically for business needs, such as
+    organization support with team management, single-sign on, and audit logging.
```

Comment:
```suggestion
    Enjoy additional functionality specifically for companies, such as
    organization support with team management, single-sign on, and audit logging.
```

Reply:
Definitely nit-picking this one :)

As a native English speaker, how do you read the word "company"?

I read it as a legal entity that is exclusionary to education, NGOs etc. But they can actually benefit from business features.

Reply:
Hrm, good point. But `business needs` is an odd phrasing that I was trying to remove.

I'd usually use something like `larger organizations`, but that was awkward with the follow sentence, but that can perhaps be fixed?

```
    Enjoy additional functionality specifically for larger organizations, such as
    team management, single-sign on, and audit logging.
```